docs/UserGuide.md

```diff
@@ -253,31 +326,26 @@ Examples:
 * `list` followed by, `unfavourite 1` unfavourites the 1<sup>st</sup>
   company in Tinner.
 
+[Back to Table of Contents](#toc)
+
 ## Modifying Roles <a id="c-modifying-roles"></a>
 
 ### Adding an internship role to an existing company: `addRole` <a id="c-add-c-r"></a>
 
 Adds an internship role to a company that already exists in the list.
 
-Format: `addRole COMPANY_INDEX n/ROLE_NAME [(TYPE)] s/STATUS [r/REMINDER_DATE] [d/DESCRIPTION] [$/STIPEND]`
-* Add internship role at the specified `COMPANY_INDEX`.
-* The `COMPANY_INDEX` must be a positive integer like 1, 2, 3, ...
-* The `ROLE_NAME` should only contain alphanumeric characters, spaces and an optional pair of round brackets.
-* The `STATUS` accepted are as follows: applying, pending, interview and assessments, offered, rejected, complete.
-* The `REMINDER_DATE` should be in format dd-MM-yyyy HH:mm.
-* The `DESCRIPTION` and `STIPEND` fields are optional during the initial role creation.
-    * The `DESCRIPTION` can contain alphanumeric characters, spaces and special characters.
-    * The `STIPEND` must be a positive integer going up to 10 digits long.
+Format: `addRole COMPANY_INDEX n/ROLE_NAME [(TYPE)] s/STATUS r/REMINDER_DATE [d/DESCRIPTION] [$/STIPEND]`
```

Comment:
Hi, I think the reminder_date should be optional as well.

Reply:
Thanks. Noted and rectified.